straxen/misc.py

```diff
@@ -41,7 +42,9 @@ def do_round(x):
 
 
 @export
-def print_versions(modules=('strax', 'straxen', 'cutax'), return_string=False):
+def print_versions(modules=('strax', 'straxen', 'cutax'),
+                   return_string=False,
+                   include_git_details=False):
```

Comment:
I have not seen the message, but if it is not too long I would even print it by default.

Reply:
Certainly possible, let me reduce the verbosity a bit such that it fits. This will be the output:
```
Working on DESKTOP-F4PI41P.localdomain with the following versions and installation paths:
python  v3.9.5  (default, Jun  4 2021, 12:28:51) [GCC 7.5.0]
strax   v1.0.0  /mnt/c/google_drive/PhD-master/ubuntu-storage/ubuntu-windows/software/strax/strax       git branch:print_git_hash | d0ad37f
utilix  v0.6.1  /home/joran/miniconda3/lib/python3.9/site-packages/utilix
```